src/bp/core/config/config-loader.ts

```diff
@@ -180,4 +180,32 @@ export class ConfigProvider {
       throw new FatalError(e, `Error reading configuration file "${fileName}"`)
     }
   }
+
+  public async getBrandingConfig(appName: 'admin' | 'studio') {
+    const defaultConfig = {
+      admin: {
+        title: 'Botpress Admin Panel',
+        favicon: 'assets/ui-admin/public/favicon.ico',
+        customCss: ''
+      },
+      studio: {
+        title: 'Botpress Studio',
+        favicon: 'assets/ui-studio/public/img/favicon.png',
+        customCss: ''
+      }
+    }
+
+    if (!process.IS_PRO_ENABLED) {
+      return defaultConfig[appName]
```

Comment:
can we warn the user that his theme won't ne applied if he has defined a custom template but not activated pro.

+ in all those "Please activate pro to enable this feature" logs we should point to sales form or something